src/iOS/Xamarin.iOS/Samples/Layers/ShowPopup/ShowPopup.cs

```diff
@@ -0,0 +1,130 @@
+﻿// Copyright 2020 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using ArcGISRuntime;
+using Esri.ArcGISRuntime.Data;
+using Esri.ArcGISRuntime.Geometry;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Mapping.Popups;
+using Esri.ArcGISRuntime.Toolkit.UI.Controls;
+using Esri.ArcGISRuntime.UI.Controls;
+using Foundation;
+using System;
+using System.Linq;
+using UIKit;
+
+namespace ArcGISRuntimeXamarin.Samples.ShowPopup
+{
+    [Register("ShowPopup")]
+    [ArcGISRuntime.Samples.Shared.Attributes.Sample(
+        name: "Show a popup",
+        category: "Layers",
+        description: "Show predefined popups from a web map.",
+        instructions: "Tap on the features to prompt a popup that displays information about the feature. ",
+        tags: new[] { "feature", "feature layer", "popup", "web map" })]
+    [ArcGISRuntime.Samples.Shared.Attributes.OfflineData()]
```

Comment:
Don't forget to remove the unused `OfflineData` attributes (note: probably worth fixing this in the sample generator script)